cli/dcoscli/data/help/package.txt

```diff
@@ -68,6 +71,9 @@ Options:
         Print a short description of this subcommand.
     --options=<file>
         Path to a JSON file that contains customized package installation options.
+    --output-directory=<output-directory>
+        Path to the directory where the data should be stored.
+        Defaults to target/ underneath the directory of the DC/OS Package Build Definition.
```

Comment:
target is very scala-y. `dcos node diagnostics` defaults to cwd if none specified. i would do the same here.